assets/product-images-slider.js

```diff
@@ -0,0 +1,91 @@
+const numberOfThumbnails = document.querySelector('[data-glide-thumbnail-slides]').dataset.glideThumbnailSlides;
```

Comment:
Carefull here, you don't have any error redundancy.

Reply:
If I google "error redundancy", it says that it means the same data is used twice using different wording.  I assumed that you meant to check that the variable is not storing something falsey, but please let me know if I am wrong.